quarkchain/cluster/tests/test_grpc_client.py

```diff
@@ -0,0 +1,106 @@
+import grpc
+import grpc_testing
+import unittest
+
+from concurrent import futures
+from quarkchain.generated import grpc_pb2
+from quarkchain.generated import grpc_pb2_grpc
+from quarkchain.cluster.grpc_client import GrpcClient
+from grpc.framework.foundation import logging_pool
+
+
+class StatusCode0(grpc_pb2_grpc.ClusterSlaveServicer):
+    def SetRootChainConfirmedBlock(self, request, context):
+        return grpc_pb2.SetRootChainConfirmedBlockResponse(
+            status=grpc_pb2.ClusterSlaveStatus(code=0, message="Confirmed")
+        )
+
+
+class StatusCode1(grpc_pb2_grpc.ClusterSlaveServicer):
+    def SetRootChainConfirmedBlock(self, request, context):
+        return grpc_pb2.SetRootChainConfirmedBlockResponse(
+            status=grpc_pb2.ClusterSlaveStatus(code=1, message="Confirmed")
+        )
+
+
+class TestGrpcClient(unittest.TestCase):
+    def setUp(self):
+        self.execution_thread = logging_pool.pool(1)
+        self.real_time = grpc_testing.strict_real_time()
+        self.real_time_test_channel = grpc_testing.channel(
+            grpc_pb2.DESCRIPTOR.services_by_name.values(), self.real_time
+        )
+
+    def shutDown(self):
+        self.execution_thread.shutdown(wait=True)
+
+    def build_test_server(self, statuscode, port):
```

Comment:
what's a `statuscode`? is it a server instance?

Reply:
like normalserver and errorserver